stor/utils.py

```diff
@@ -312,6 +313,52 @@ def is_writeable(path, swift_retry_options=None):
     return answer
 
 
+def validate_file_path(dest, _warn=False):
```

Comment:
nit: `dest` name implies this is always a check on destination path but it's called on both destination and source paths.